### DISCUSSION TITLE
P1001 connection fail

### DISCUSSION THREAD

**Discussion by chengyang9966** [prisma/prisma#21420] [Q&A]
### Question

 dotenv -e .env.uat -- npx prisma migrate dev
```
  prisma:engines  binaries to download libquery-engine, migration-engine, introspection-engine, prisma-fmt +0ms
  prisma:loadEnv  project root found at /root/MngCredit/mng-cep/package.json +0ms
  prisma:tryLoadEnv  Environment variables not found at null +0ms
  prisma:tryLoadEnv  Environment variables not found at undefined +0ms
  prisma:tryLoadEnv  No Environment variables loaded +0ms
  prisma:getConfig  Using getConfig Wasm +0ms
  prisma:getConfig  config data retrieved without errors in getConfig Wasm +1ms
  prisma:loadEnv  project root found at /root/MngCredit/mng-cep/package.json +4ms
  prisma:tryLoadEnv  Environment variables not found at null +2ms
  prisma:tryLoadEnv  Environment variables not found at undefined +1ms
  prisma:tryLoadEnv  No Environment variables loaded +0ms
Prisma schema loaded from src/prisma/schema.prisma
  prisma:getConfig  Using getConfig Wasm +1ms
  prisma:getConfig  config data retrieved without errors in getConfig Wasm +1ms
Datasource "db": MySQL database "mng_cep_dev" at "locolhost:3306"

  prisma:getDMMF  Using CLI Query Engine (Node-API Library) at: /root/MngCredit/mng-cep/node_modules/@prisma/engines/libquery_engine-rhel-openssl-1.1.x.so.node +0ms
  prisma:getDMMF  Loaded Node-API Library +7ms
  prisma:getDMMF  unserialized dmmf result ready +20ms
  prisma:getDMMF  dmmf retrieved without errors in getDmmfNodeAPI +11ms
  prisma:getConfig  Using getConfig Wasm +70ms
  prisma:getConfig  config data retrieved without errors in getConfig Wasm +1ms
  prisma:getConfig  Using getConfig Wasm +0ms
  prisma:getConfig  config data retrieved without errors in getConfig Wasm +1ms
Error: Error: P1001: Can't reach database server at `locolhost`:`3306`

Please make sure your database server is running at `locolhost`:`3306`.
    at ensureDatabaseExists (/root/MngCredit/mng-cep/node_modules/prisma/build/index.js:93119:11)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async _MigrateDev.parse (/root/MngCredit/mng-cep/node_modules/prisma/build/index.js:95363:26)
    at async main (/root/MngCredit/mng-cep/node_modules/prisma/build/index.js:98674:18)
```

i am using localhost mariadb 
when i test the connection using script i am able to connect 
when i am using prisma migrate then hit connection issue 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @chengyang9966 👋 

Thank you for your question.

You may have a typo in your code. You probably meant to write `localhost` but you wrote `locolhost`.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
